setup.py

```diff
@@ -7,36 +7,34 @@
 __version__ = VERSION
 __versionstr__ = ".".join(map(str, VERSION))
 
-f = open(join(dirname(__file__), "README"))
-long_description = f.read().strip()
-f.close()
+with open(join(dirname(__file__), "README")) as f:
+    long_description = f.read().strip()
 
 install_requires = ["urllib3>=1.21.1"]
 tests_require = [
     "requests>=2.0.0, <3.0.0",
     "nose",
     "coverage",
     "mock",
-    "pyaml",
+    "pyyaml",
     "nosexcover",
 ]
 
 docs_require = ["sphinx<1.7", "sphinx_rtd_theme"]
 generate_require = ["black", "jinja2"]
 
-# use external unittest for 2.6
-if sys.version_info[:2] == (2, 6):
-    install_requires.append("unittest2")
-
 setup(
     name="elasticsearch",
     description="Python client for Elasticsearch",
-    license="Apache License, Version 2.0",
+    license="Apache-2.0",
```

Comment:
Changing this field to the SPDX identifier for the license as is typically recommended.